toolkit/scripts/build_go_vendor_cache.sh

```diff
@@ -2,8 +2,92 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+# Quit on failure
```

Comment:
Please verify that this isn't really used anymore so that we can take it over this way.

Reply:
Verified that it is not used by anything